content/en/agent/logs/proxy.md

```diff
@@ -60,17 +69,219 @@ The parameter above can also be set with the following environment variable:
 * `DD_LOGS_CONFIG_SOCKS5_PROXY_ADDRESS`
 
 {{% /tab %}}
-{{% tab "HTTPS" %}}
+{{< /tabs >}}
+
+## Examples of TCP proxy
+
+### Using HAProxy as a TCP Proxy for Logs
+
+This example explains how to configure the Datadog Agent to send logs in TCP to a server with HAProxy installed and listening on port `10514` to then forward the logs to Datadog.
+
+`agent ---> haproxy ---> Datadog`
+
+The encryption is disabled between the Agent and HAProxy which is then configured to encrypt the data before sending it to Datadog.
+
+#### HAProxy configuration
+
+HAProxy should be installed on a host that has connectivity to Datadog. Use the following configuration file if you do not already have it configured.
+
+{{< tabs >}}
+{{% tab "Datadog US site" %}}
+
+```conf
+# Basic configuration
+global
+    log 127.0.0.1 local0
+    maxconn 4096
+    stats socket /tmp/haproxy
+
+# Some sane defaults
+defaults
+    log     global
+    option  dontlognull
+    retries 3
+    option  redispatch
+    timeout client 5s
+    timeout server 5s
+    timeout connect 5s
+
+# This declares a view into HAProxy statistics, on port 3833
+# You do not need credentials to view this page and you can
+# turn it off once you are done with setup.
+listen stats
+    bind *:3833
+    mode http
+    stats enable
+    stats uri /
+    
+# This declares the endpoint where your Agents connects for
+# sending Logs (e.g the value of "logs.config.logs_dd_url")
+frontend logs_frontend
+    bind *:10514
+    mode tcp
+    default_backend datadog-logs
+    
+# This is the Datadog server. In effect any TCP request coming
+# to the forwarder frontends defined above are proxied to
+# Datadog's public endpoints.
+
+backend datadog-logs
+    balance roundrobin
+    mode tcp
+    option tcplog
+    server datadog agent-intake.logs.datadoghq.com:10516 ssl verify required ca-file /etc/ssl/certs/ca-certificates.crt check port 10516
+```
+
+**Note**: Download the certificate with the following command:
+        * `sudo apt-get install ca-certificates` (Debian, Ubuntu)
+        * `yum install ca-certificates` (CentOS, Redhat)
+The file might be located at `/etc/ssl/certs/ca-bundle.crt` for CentOS, Redhat.
```

Comment:
```suggestion
If successful, the file will be located at `/etc/ssl/certs/ca-bundle.crt` for CentOS, Redhat.
```

Reply:
Cecilia recommended wording for "might" and I think it could be applicable here. Let me know your thoughts.